Demo/Common/Configuration/ApplePaySettingsView.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import SwiftUI
+
+@available(iOS 13.0.0, *)
+internal struct ApplePaySettingsView: View {
+    @ObservedObject internal var viewModel: ConfigurationViewModel
+
+    private enum ConfigurationSection: String, CaseIterable {
+        case merchantIdentifier = "{YOUR_APPLE_PAY_MERCHANT_IDENTIFIER}"
```

Comment:
Maybe use `merchant.com.domainname.appname` like on Apple Web Site?

![Screenshot 2023-08-14 at 13 26 16](https://github.com/Adyen/adyen-ios/assets/2648655/dfad973d-8447-4487-9e1c-190609aece11)

Reply:
Does it have to be `case` ? can we just use `static let` ?

Reply:
yes we can add static let :) I will change it. :)

Reply:
c48ebc09d2550f9e7e445b4a42111155958e0b9a